Aptoide/appcoins-wallet-android
# [APPC-1118] List local payments methods

**What does this PR do?**

  Refactors previous hardcoded list of payment, so that it now receives any number of payment methods.
  Adds the new payment methods view logic
  Changes layout to have scrollview to better accommodate the new number of payment methods

**Where should the reviewer start?**

PaymentMethodFragment.java
payment_methods_layout.xml

**How should this be manually tested?**

 Trivial Drive -> Buy gas -> Check if all payments are there (3 new, alfamart, bank, guppy). Check if the bonus appears on the correct payment methods (it should appear on the new ones). Check if anything breaks
Note: The handle of buy button press when the 3 new payments are selected should not do anything and are supposed to be done in another ticket (1102)

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1118
(implementation of new payments ) https://aptoide.atlassian.net/browse/APPC-1102

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
